pythoncz/models/github.py

```diff
@@ -42,9 +42,12 @@ def _create_api_session():
                   '(+https://python.cz)').format(now=datetime.now())
 
     session = requests.Session()
+    token = app.config['GITHUB_TOKEN']
+    if not token:
+        raise Exception('GITHUB_TOKEN not configured')
```

Comment:
ValueError?

Reply:
Nevím... Tak úplně na popis ValueError to nesedí, a navíc to ničemu nepomůže. Možná tak vytvořit nějakou ConfigurationError.